CHANGES.rst

```diff
@@ -53,6 +53,8 @@ API changes
     ``BasicPSFPhotometry``, ``IterativelySubtractedPSFPhotometry`` and
     ``DAOPhotPSFPhotometry``. [#745]
 
+  - Added method ``_compute_local_model`` to ``GriddedPSFModel``. [#1012]
+
```

Comment:
Changelog entry is not needed for non-user facing issues

Reply:
Removed entry